codebuild_specs/e2e_workflow.yml

```diff
@@ -1417,95 +1404,110 @@ batch:
         - publish_to_local_registry
-    - identifier: amplify_table_3
+    - identifier: 3_gsis_100k_records
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
-          TEST_SUITE: src/__tests__/amplify-table-3.test.ts
-          CLI_REGION: eu-west-1
+          TEST_SUITE: src/__tests__/deploy-velocity/3-gsis-100k-records.test.ts
+          CLI_REGION: sa-east-1
       depend-on:
         - publish_to_local_registry
-    - identifier: amplify_table_2
+    - identifier: replace_2_gsis_100k_records
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
-          TEST_SUITE: src/__tests__/amplify-table-2.test.ts
-          CLI_REGION: eu-west-2
+          TEST_SUITE: src/__tests__/deploy-velocity/replace-2-gsis-100k-records.test.ts
+          CLI_REGION: us-east-1
       depend-on:
         - publish_to_local_registry
-    - identifier: amplify_table_1
+    - identifier: replace_2_gsis_update_attr_100k_records
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
-          TEST_SUITE: src/__tests__/amplify-table-1.test.ts
-          CLI_REGION: eu-west-3
+          TEST_SUITE: >-
+            src/__tests__/deploy-velocity/replace-2-gsis-update-attr-100k-records.test.ts
+          CLI_REGION: us-east-2
       depend-on:
         - publish_to_local_registry
-    - identifier: amplify_ddb_canary
+    - identifier: single_gsi_100k_records
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
-          TEST_SUITE: src/__tests__/amplify-ddb-canary.test.ts
-          CLI_REGION: me-south-1
+          TEST_SUITE: src/__tests__/deploy-velocity/single-gsi-100k-records.test.ts
+          CLI_REGION: us-west-1
       depend-on:
         - publish_to_local_registry
-    - identifier: all_auth_modes
+    - identifier: dynamic_group_auth_ddbprimary_ddbrelated
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
-          TEST_SUITE: src/__tests__/all-auth-modes.test.ts
-          CLI_REGION: sa-east-1
+          TEST_SUITE: >-
+            src/__tests__/group-auth/dynamic-group-auth/dynamic-group-auth-ddbprimary-ddbrelated.test.ts
+          CLI_REGION: us-west-2
       depend-on:
         - publish_to_local_registry
-    - identifier: admin_role
+    - identifier: dynamic_group_auth_ddbprimary_sqlrelated
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
-          TEST_SUITE: src/__tests__/admin-role.test.ts
-          CLI_REGION: us-east-1
+          TEST_SUITE: >-
+            src/__tests__/group-auth/dynamic-group-auth/dynamic-group-auth-ddbprimary-sqlrelated.test.ts
+          CLI_REGION: ap-east-1
```

Comment:
Note: this test fails in ap-east-1 because Cognito User Pools is not supported in this region. The E2E run maps it to a different region, and I will follow up with a separate PR to ensure tests that require Cognito do not run in this region.